modules/sensors/hotswap.c

```diff
@@ -137,7 +137,12 @@ void vTaskHotSwap( void *Parameters )
             continue;
         }
 
-        if ( new_state_amc ^ old_state_amc ) {
+	bool standalone_mode = false;
+        if (get_ipmb_addr() == 0xA2) {
```

Comment:
I feel like the `0xA2` constant is being used in enough places here that it should be made into a macro. That way, its meaning is clearer, and you avoid having magic numbers around the code.